### ISSUE TITLE
Some issues with import export

### ISSUE THREAD

**Issue by kpcee** [flameshot-org/flameshot#1358]
I found some minor bugs with import and export:

The file requester always uses the directory of the binary and not the home directory as default directory. The configuration is saved as ini file, the exported file as conf (not really a problem, only if you want to exchange the files manually you have to rename them before).
But not all variables are read during import, this affects at least the options

* Confirmation required to delete screenshot from the latest uploads
* Automatic check for updates
* Show welcome message on launch
* Use fixed path for screenshots to save
* (Shortcuts)

Git version (commit 16fd3b2915829ccb550a023d7b829b7a46beebde)
Arch Linux, Xfce

**Comment by mmahmoudian** [comment]
@kpcee considering that @veracioux refactored some part of the code to make the config error catching, I wonder if the issue is still there.

**Comment by kpcee** [comment]
I can't currently test it because I've already migrated to Qt 6.2 and Flameshoot don't compile with Qt 6, it uses QDesktopWidget (deprecated in Qt 5, and has been removed in Qt 6) and maybe other things.

**Comment by mmahmoudian** [comment]
@kpcee yeah, we are still on Qt 5 and perhaps for the next version we will migrate to Qt 6 if I remember correctly from the discussions.
